terminal/bundles/org.eclipse.terminal.view.core/src/org/eclipse/terminal/view/core/ITerminalService.java

```diff
@@ -32,34 +33,48 @@ public interface Done {
 		public void done(IStatus status);
 	}
 
+	/**
+	 * Empty callback if no action is needed
+	 */
+	Done NOOP = new Done() {
```

Comment:
I think the while Done interface can be replaced by a `Consumer<IStatus>` instead.

I also don't thing we need something like a noop one can simply calls 

`openConsole(Map<String, Object> properties, s->{});`

or we even better add default implemented methods that do not use a "done" parameter at all.

Reply:
Thank you for the comment that is very helpful to understand the expected style of code @laeubi